packages/react-reconciler/src/ReactFiberLane.js

```diff
@@ -370,7 +319,7 @@ export function getNextLanes(root: FiberRoot, wipLanes: Lanes): Lanes {
 
   // If there are higher priority lanes, we'll include them even if they
   // are suspended.
-  nextLanes = pendingLanes & equalOrHigherPriorityLanes;
+  nextLanes = pendingLanes & getEqualOrHigherPriorityLanes(nextLanes);
```

Comment:
Is it intentional that the algorithm for determining **the equal or higher priority lanes** also changes? To be specific, if the lanes for some lane priority are `0b111` and `nextLanes` equals to `0b010`, `equalOrHigherPriorityLanes` would be `0b111` while `getEqualOrHigherPriorityLanes(nextLanes)` would be `0b011`. I think that's why we had to tweak the test below.